asteroid_data_processing.py

```diff
@@ -0,0 +1,417 @@
+
+app = Flask(__name__)
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+# logging.getLogger().setLevel(logging.WARNING)
+
+
+class bcolors:
+    HEADER = "\033[95m"
+    OKBLUE = "\033[94m"
+    OKCYAN = "\033[96m"
+    OKGREEN = "\033[92m"
+    WARNING = "\033[93m"
+    FAIL = "\033[91m"
+    ENDC = "\033[0m"
+    BOLD = "\033[1m"
+    UNDERLINE = "\033[4m"
+
+
+def timeit(func):
+    def timed(*args, **kwargs):
+        ts = time.time()
+        result = func(*args, **kwargs)
+        te = time.time()
+        logging.info(f"Function: {func.__name__} time: {round((te -ts)*1000,1)} ms)")
+        return result
+
+    return timed
+
+
+@timeit
+def read_config():
+    config_path = "config.ini"
+    try:
+        config = configparser.ConfigParser()
+        config.read(config_path)
+
+        RADIUS_FACTOR = config.getfloat("Parameters", "RADIUS_FACTOR")
+
+        MAX_STARS = config.getint("Parameters", "MAX_STARS")
+
+        MAX_DISTANCE = config.getfloat("Parameters", "MAX_DISTANCE")
+
+        QUERY_STARS_LIMIT = config.getfloat("Parameters", "QUERY_STARS_LIMIT")
+
+        MIN_DEG = config.getfloat("Parameters", "MIN_DEG")
+
+    except FileNotFoundError:
+        logging.exception("There is no config file named config.ini")
+    return RADIUS_FACTOR, MAX_STARS, MAX_DISTANCE, QUERY_STARS_LIMIT, MIN_DEG
+
+
+@timeit
+def separate_data(api_response, name, location):
+    match = re.search(r"\$\$SOE.*?\$\$EOE", api_response.text, re.DOTALL)
+    if match:
+        fragment = match.group()
+        lines = fragment.strip().split("\n")
+        asteroid_positions = []
+
+        for line in lines:
+            line = line.strip()
+            parts = line.split()
+            if len(parts) >= 2:
+                date_time = parts[0] + " " + parts[1]
+                datetime_obj = datetime.strptime(date_time, "%Y-%b-%d %H:%M")
+
+                # Tworzenie obiektu Time z obiektu datetime
+                time = Time(datetime_obj)
+                ra = parts[2] + " " + parts[3] + " " + parts[4]
+                dec = parts[5] + " " + parts[6] + " " + parts[7]
+                ra = better_pos_ra(ra)
+                dec = better_pos_dec(dec)
+                alt = get_altitude(ra, dec, time, location)
+                asteroid_positions.append([name, time, ra, dec, alt])
+        return np.array(asteroid_positions)
+    else:
+        logging.exception("No ephemeris for target")
+        return []
+
+
+def better_pos_ra(pos):
+    parts = pos.split()
+    hours = parts[0]
+    minutes = parts[1]
+    seconds = parts[2]
+
+    formatted_pos = f"{hours}h{minutes}m{seconds}s"
+    return formatted_pos
+
+
+def better_pos_dec(pos):
+    parts = pos.split()
+    hours = parts[0]
+    minutes = parts[1]
+    seconds = parts[2]
+
+    formatted_pos = f"{hours}d{minutes}m{seconds}s"
+    return formatted_pos
+
+
+def get_radius(x, y, x_mean, y_mean, RADIUS_FACTOR):
```

Comment:
param names should use lower-case
```suggestion
def get_radius(x, y, x_mean, y_mean, radius_factor):
```